hail/python/hailtop/utils/utils.py

```diff
@@ -116,7 +116,7 @@ def grouped(n: int, ls: List[Any]) -> Iterable[Any]:
         yield group
 
 
-def partition(k, ls):
+def partition(k: int, ls: List[int]) -> Iterable[int]:
```

Comment:
`ls` doesn't have to contain integers only. It can be any kind of value. So for `ls` and the return type (but not `k`) replace `int` with `Any`.